src/python/build_management/build_manager.py

```diff
@@ -1377,12 +1416,25 @@ def setup_build(revision=0):
   if system_binary:
     return setup_system_binary()
 
+  fuzz_target_build_bucket_path = environment.get_value(
```

Comment:
Was thinking if we could reuse `get_primary_bucket_path` here somehow, but don't see a way given that fuzz target is not chosen yet and the order of preference here is different.

Reply:
Yes, the use case is a little different here, so I don't think we can re-use it here.